### DISCUSSION TITLE
Expense Accounts level of detail

### DISCUSSION THREAD

**Discussion by moritzj29** [firefly-iii/firefly-iii#5302] [General]
Hello,
I'm quite new to firefly iii and try to grasp all of the concepts. I've done accounting before, but with much simpler tools.

My question arises around the expense accounts. Using an importer, these are mostly setup automatically or matched to existing accounts. Which level of detail do you keep in your expense accounts?

E.g. I setup expense accounts for different grocery stores (e.g. Aldi and Lidl). I have several transactions in each shop every month and this way I can break it down to a shop level. Nice.

On the other hand, I set up a "general" pharmacy expense account. I don't go to the pharmacy very often and it is a different pharmacy every time. So instead of having 6 pharmacy accounts for 6 transactions, I only have one general pharmacy account. I don't care at which pharmacy I spent the money. Similarly with take-out food or restaurants, here the shop is also different most of the time (depends on your habits).

So my question is, do you set up individual expense accounts for every shop and group them together with categories? Or do you already group into more general expense accounts? According to the documentation, the first approach is the suggested one. But it will easily result in hundreds or thousands of shops (=expense accounts) for people like me who pay everything with Apple Pay and not with cash...

Of course this depends on personal preferences, but I'would like to hear some opinions on the topic. Maybe I have missed some points worth considering?

Thanks everyone for your ideas!

old reddit discussion for reference: https://www.reddit.com/r/FireflyIII/comments/lthpo5/question_regarding_making_destination_accounts/
additional issues/discussion references are also welcome!

**Comment by niklas2810** [comment]
For me it depends. As you already said, I sometimes am very precise with the expense account name (e.g. "bakery xy"), but sometimes I also use a very unspecific expense account (like "Amazon", even though I order at the Samsung shop for example).

And I think there really is no "correct" and "wrong" way to set your accounts up! The priority for me is to always be able to understand where I'm spending most of my money. And if I see that 200 bucks go to "Amazon", I don't need to know to which seller exactly. However, I could also use a tag called "Amazon" and specify the expense account for more details!

You can sometimes decide whether to use categories, expense accounts or tags, you can model your use case with all three of them :D What counts is that you can read from the data what you want to know!

**Reply by moritzj29** [reply]
even if some time has passed already, I just wanted to thank you for your reply! It was a great help at the time, eventhough I did not stick to firefly-iii for my bookkeeping at the end

**Comment by trallnag** [comment]
Good topic. I'm also wondering about the best way to approach it. @moritzj29 mentioned only having a single account for pharmacy stuff. Now shouldn't that be a category and not an account?

**Reply by moritzj29** [reply]
Actually I moved away from firefly-iii due to it's complexity and very fix concepts in accounts, categroies, budgets, tags, ...
For me, double-entry bookkeeping accounts make more sense if they are based on categories, e.g. clothing. At the end of the day I don't care if bought items in shop A or B.
Then I can add or remove money from this account. This is not possible in firefly, since income and expense accounts are strictly separated. E.g. how do you refunds when you return something? It's not income. It actually reduces your expenses.
In double-entry bookkeeping you usually do not require categories, since everything should be deductible from the account structure.
But it depends on your preferences. firefly has a very nice UI and powerful automation and import features. You just have to follow it's concepts very strictly.

**Reply by trallnag** [reply]
What do you use now? I'm using "Mein Haushaltsbuch" and thinking about alternatives. It's simply and nice to use but Android only.

**Reply by moritzj29** [reply]
I'm currently experimenting with the "nerdy" approach using `beancount`, which is accounting based on plaintext files. beancount is written in Python but there are several very similar alternatives in different languages. It also has a web interface called `fava`, but much simpler than firefly-iii
